### PR TITLE
Rename the `assertSecondEqual` utility method to `assertInDelta`

### PR DESCRIPTION
**Pull request by illia-shepilov**
This PR renames the `assertSecondEqual` utility method to `assertInDelta`.

### CODE REVIEW

**Comment by illia-shepilov** [comment]
@armiol PTAL

File: testlib/src/main/java/io/spine/testing/Tests.java [hunk 1]
```diff
@@ -171,15 +171,17 @@ public static void assertMatchesMask(Message message, FieldMask fieldMask) {
...
+     * than the set delta.
      *
-     * @param expectedSec expected timestamp value
-     * @param actualSec   actual timestamp value
-     * @param maxDiffSec  the maximum expected difference between the values
+     * <p>The assertion will be passed if the actual delta equals to the set one.
+     *
+     * @param expectedValue expected value
+     * @param actualValue   actual value
+     * @param delta  the maximum expected difference between the values
```
Comment (armiol) [inline testlib/src/main/java/io/spine/testing/Tests.java]: I think this line isn't aligned.

File: testlib/src/test/java/io/spine/testing/TestsShould.java [hunk 2]
```diff
@@ -217,22 +217,22 @@ private long now() {
...
-            assertSecondsEqual(recentTime, recentTime, 0);
+            assertInDelta(recentTime, recentTime, 0);
         }
 
         @Test
         @DisplayName("when values are close")
         void closeValues() {
             // This method would be called within 10 seconds.
-            Tests.assertSecondsEqual(recentTime, now(), 10);
+            Tests.assertInDelta(recentTime, now(), 10);
```
Comment (armiol) [inline testlib/src/test/java/io/spine/testing/TestsShould.java]: Not sure why you still use time. Please use regular `long` values. And there are way more cases:

* `expected` == `actual`;
* `expected` < `actual` and `actual` < (`expected` + delta);
* `expected` < `actual` and `actual`  == (`expected` + delta); 
* `expected` < `actual` and `actual`  >  (`expected` + delta) — negative scenario; 
* `expected` > `actual` and `actual` > (`expected` - delta);
* `expected` > `actual` and `actual`  == (`expected` - delta); 
* `expected` > `actual` and `actual`  < (`expected` - delta) — negative scenario.

**Comment by illia-shepilov** [comment]
@armiol PTAL

File: testlib/src/test/java/io/spine/testing/TestsShould.java [hunk 3]
```diff
@@ -199,40 +199,83 @@ void fieldIsNotSet() {
...
+        @Test
+        @DisplayName("when the actual value equals the sum of the expected value and delta")
+        void actualValueEqualsTheSumExpectedValueAndDelta() {
+            long expectedValue = getValue();
+            long actualValue = expectedValue + DELTA;
+            assertInDelta(expectedValue, actualValue, DELTA);
+        }
+
+        @Test
+        @DisplayName("throw when the actual value grater than the sum of the expected value and delta")
```
Comment (armiol) [inline testlib/src/test/java/io/spine/testing/TestsShould.java]: `greater`. It's the same copy-paste everywhere.

File: testlib/src/test/java/io/spine/testing/TestsShould.java [hunk 4]
```diff
@@ -199,40 +199,83 @@ void fieldIsNotSet() {
...
+        @Test
+        @DisplayName("throw when the actual value less than the subtraction of the expected value and delta")
+        void actualValueLessThanTheSubtractionExpectedValueAndDelta() {
+            long actualValue = getValue();
+            long expectedValue = actualValue + DELTA + 1;
             assertThrows(
                     AssertionError.class,
-                    () -> Tests.assertSecondsEqual(100, 200, 2)
+                    () -> assertInDelta(expectedValue, actualValue, DELTA)
+
```
Comment (armiol) [inline testlib/src/test/java/io/spine/testing/TestsShould.java]: I think that's a redundant line.

**Comment by illia-shepilov** [comment]
@armiol PTAL

**Pull request merged**